### PR TITLE
Added PlantSnap

### PR DESCRIPTION
**Pull request by OfficialMuffin**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -5469,7 +5469,17 @@
         "notes_de": "Schreibe dem Support per E-Mail und bitte um eine Löschung des Accounts",
         "email": "service@pixum.com"
     },
-
+
```
Comment (tupaschoal) [inline _data/sites.json]:
```suggestion

```

File: _data/sites.json [hunk 2]
```diff
@@ -5469,7 +5469,17 @@
...
+    {
+        "name": "PlantSnap",
+        "url": "https://www.plantsnap.com/support-center/get-in-touch/",
+        "difficulty": "medium",
+        "notes": "To delete your account, open the app on your mobile device and click the \"More\" option button. Then click your profile name, from there you will be able to delete your account. If you require additional help, use the contact form provided on their website.",
+        "domains": [
+            "plantsnap.com"
+        ]
+    },
+
```
Comment (tupaschoal) [inline _data/sites.json]:
```suggestion

```

**Comment by tupaschoal** [comment]
Looks fine, once the whitespaces are removed

**Comment by OfficialMuffin** [comment]
I'm using the web editor to add PlantSnap. By doing this its adding whitespace. But previewing the code looks fine and all applies by the contrib rules.
I struggle with using git to commit the PR

Sorry

Thanks,
Leon

________________________________

**Comment by tupaschoal** [comment]
Accepting the changes I suggested should remove the whitespaces, I  would've changed them myself, but you didn't allow changes from maintainers. It's an easy fix nonetheless

**Comment by tupaschoal** [comment]
Do note that to see and commit the suggestions I made you either have to be in the desktop or "view as in desktop" on a mobile browser.

**Comment by Nightfirecat** [comment]
I've fixed the whitespace errors and pushed this to master in commit 81e587bf8f9f43e9864ce9123e9bca41e665beb4. Thanks for your contribution!

**Pull request closed without merging**